INSTALL.md

```diff
@@ -143,10 +137,7 @@ directory for this repository.
 
 Then compile and run the following (after ensuring you have installed faiss):
 
-```
-make demos
-./demos/demo_sift1M
-```
+** how to compile demo_sift1M **
```

Comment:
`$ make -C build demo_sift1M`

Reply:
Does not work due to MKL linking

https://gist.github.com/mdouze/975b681c1a45f03664851d646a6f4243

Reply:
`$ cmake -B build -DBLA_VENDOR=Intel10_64_dyn -DFAISS_ENABLE_GPU=OFF .`
`$ make -C build -j demo_sift1M`
works fine for me. There may be an issue with your MKL installation?